Documentation meta-descriptions review N° 3 (#5547)
wazuh/wazuh-documentation: source/user-manual/uninstall/elastic-stack.rst

.. Copyright (C) 2015, Wazuh, Inc.

.. meta::

>   :description: Check out how to uninstall Wazuh with Elastic Stack. This section of our documentation will give instructions to uninstall the Wazuh components.

Reviewer: ```suggestion
  :description: Check out how to uninstall Wazuh with Elastic Stack in this section of our documentation.
```
